contact/pubspec.lock

```diff
@@ -531,8 +531,8 @@ packages:
     dependency: "direct main"
     description:
       path: "."
-      ref: master
-      resolved-ref: a1b79695b844888ec8330d41964d1a054c348bc2
+      ref: cnb_support
+      resolved-ref: "351a1bf0fef48f59d771d3a485cbada950f2ed0a"
```

Comment:
PR [jmap-dart-client](https://github.com/linagora/jmap-dart-client/pull/78) has been merged into master. Please update it.

Reply:
because in `jmap-dart-client` we also had some other fixed not merged in customer, so I temporarily create a dedicated branch for customer